builder-base/scripts/generate_daemon_image

```diff
@@ -58,7 +58,7 @@ fi
 # means that we'll skip committing the container
 
 # get the delta
-delta=`rsync -ani --size-only --omit-dir-times --exclude=base --out-format='%n' $CEPH_INSTALL_PATH/ $CEPH_INSTALL_PATH/base`
+delta=`rsync -ani --omit-dir-times --exclude=base --out-format='%n' $CEPH_INSTALL_PATH/ $CEPH_INSTALL_PATH/base`
```

Comment:
When the image is already built and we want to re-compile and build a new image, this command returns:

```
delta=
```

But removing `--size-only` we re-copy all the binaries but I feel like this is a small price to pay... Unless it creates a  layering problem where diffs of our Docker images in our registry can't be pulled easily anymore.